### PR TITLE
Adding Component Folder

### PR DESCRIPTION
**Pull request by asrivastava0308**
task-378405

Add Component Folder And readme 

Note : Can't add index.scss file under components as the current lint process don't let the build to pass .

### CODE REVIEW

File: css/src/components/index.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@import '';
```
Comment (wibjorn) [inline css/src/components/index.scss:1]: Can the file remain empty while satisfying lint? If not, we can remove it and add a README.md

File: .changeset/honest-queens-listen.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
```
Comment (wibjorn) [inline .changeset/honest-queens-listen.md:2]: `patch`

File: css/src/components/index.scss (hunk 1 above)
Comment (wibjorn) [reply css/src/components/index.scss:1]: Readme could be populated with some of details from the plan: 
https://github.com/microsoft/atlas-design/pull/2

Comment (asrivastava0308) [reply css/src/components/index.scss]: Current lint process not allowing the file to remain empty . So removed it

File: .changeset/honest-queens-listen.md (hunk 2 above)
Comment (asrivastava0308) [reply .changeset/honest-queens-listen.md]: changed it to patch , as no changeset create an warning

File: css/src/components/component.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#### Components
+
+Create new component when you have a complex layout that requires many helper classes to recreate. Keep following in mind while creating a New component .
```
Comment (v-mingho) [inline css/src/components/component.md:3]: "Create a new component ......... recreate. Keep the following in mind while creating a new component."

File: css/src/components/component.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+- Component must be reusable.
+- Should be output producing.
```
Comment (v-mingho) [inline css/src/components/component.md:6]: I'm not entirely sure what this line is supposed to mean. Can it be reworded to something more simple?

Comment (wibjorn) [reply css/src/components/component.md:6]: We can probably remove 
```
- Component must be reusable.
- Should be output producing.

File: css/src/components/component.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+- File name should match the class of the component.
+- Top level classes per file: 1
```
Comment (wibjorn) [inline css/src/components/component.md:8]: There should only be one top level class per file.

**Review by wibjorn — APPROVED** [review]

**Pull request merged**